crypto/signature_cgo.go

```diff
@@ -27,10 +27,12 @@ import (
 	"github.com/ethereum/go-ethereum/crypto/secp256k1"
 )
 
+// Ecrecover returns the public key that created the given signature.
 func Ecrecover(hash, sig []byte) ([]byte, error) {
 	return secp256k1.RecoverPubkey(hash, sig)
 }
 
+// SigToPub returns the public key that created the given signature.
```

Comment:
"SigToPub returns the public key" -> "SigToPub returns the uncompressed public key"

Reply:
Compressed/uncompressed doesn't matter for SigToPub because the return value isn't encoded.